docs/manuals/core/config/job-config.md

```diff
@@ -114,4 +114,67 @@ For example, the following configuration will allow Taipy to run up to eight `Jo
     max_nb_of_workers = "8:int"
     ```
 
+# Cluster mode
+
+!!! warning "Available in Taipy Enterprise edition"
+
+    This section is relevant only to the Enterprise edition of Taipy.
+
+With the *cluster* mode, a `Job^` runs in its own environment (a worker environment), which can be
+local or on a remote machine, in an asynchronous manner. At the submission, the job is queued in a
+RabbitMQ queue. When there is a worker available, the job is dequeued and sent to the worker
+environment to be executed.
```

Comment:
Trying to add some motivation. What do you think?

```suggestion
The *cluster* mode is designed to make a Taipy application execute the jobs on a remote cluster of distributed workers. The number of machines available in the cluster is unlimited, so the application becomes fully scalable horizontally. 
With the *cluster* mode, a `Job^` asynchronously runs in its own environment. At the submission, the job is queued in a RabbitMQ queue. When there is a worker available, the job is dequeued and sent to the worker environment to be executed.
```

Reply:
I realigned the motivation sentence and push a new commit. I think this is a great addition to better explain this mode